service/version_bundles.go

```diff
@@ -12,32 +12,32 @@ func newVersionBundles() []versionbundle.Bundle {
 			Changelogs: []versionbundle.Changelog{
 				{
 					Component:   "kubernetes",
-					Description: "Enable encryption at rest.",
+					Description: "Enabled encryption at rest.",
 					Kind:        versionbundle.KindAdded,
 				},
 				{
 					Component:   "kubernetes",
-					Description: "Update to version 1.9.0.",
+					Description: "Updated to version 1.9.0.",
 					Kind:        versionbundle.KindChanged,
 				},
 				{
 					Component:   "kubernetes",
-					Description: "Use vanilla (previously coreos) hyperkube image.",
+					Description: "Used vanilla (previously coreos) hyperkube image.",
```

Comment:
For my English, this should actually be `Use vanilla`, as it refers more to the effect of the change. I know, it's not what I wrote in those simple guidelines. Seems like it's not so easy to make guidelines for writing English sentences.

Reply:
Everything refers to the effect of the change. I don't understand. You maybe wanted to make it noun like `Use of`, but let's please avoid mixing-in nouns here. 

Another links to convince you to the imperative form (beyond simple guidelines):
https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages
https://coreos.com/releases/#1576.5.0

Reply:
New suggestion in past rense:

`Changed to use vanilla (previously coreos) hyperkube image`

`Switched to vanilla (previously coreos) hyperkube image`